dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -371,6 +371,73 @@ def test_profiled_kurtosis(self):
         df = pd.concat([df1, df2, df3])
         self.assertAlmostEqual(0.3311739, num_profiler.kurtosis)
 
+    def test_bias_correction_option(self):
+        data = np.linspace(-5, 5, 11).tolist()
+        df1 = pd.Series(data)
+
+        data = np.linspace(-3, 2, 11).tolist()
+        df2 = pd.Series(data)
+
+        data = np.full((10,), 1)
+        df3 = pd.Series(data)
+
+        # Disable bias correction
+        options = FloatOptions(); options.bias_correction.is_enabled = False
+        num_profiler = FloatColumn(df1.name, options=options)
+        num_profiler.update(df1.apply(str))
+        self.assertAlmostEqual(10, num_profiler.variance)
```

Comment:
why the variance is 10? I used numpy and it returns 43.5 or did I miss something here?

Reply:
in general, I'd suggest adding some comments of how to get the value 10, or show formular to calculate it.

Reply:
The data is defined by np.linspace(-5,5,11), which forms a list of 11 equally-spaced numbers from -5 to 5 (i.e. -5, -4, -3.,...,3,4,5). I think the variance of just {-5,5,11} is 43.5 which is why you're getting that.